src/components/CardForm/CardForm.js

```diff
@@ -15,7 +15,7 @@ const CardForm = ({ colId, handleCancelButton, handleAddCard }) => {
 
     saveCard({ columnId: colId, text: newCardText }).then((newCard) => {
       handleAddCard(newCard);
-    }).catch(() => handleCancelButton());
+    });
```

Comment:
do we want a catch here? i don't know if there is anything we need to handle, but it will stop the error being logged in the console by default

Reply:
I think we need catch to resolve Promise, some way